Canada Selected Variables (3) Data only

rktakk opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## Checklist
- [ ] I can confirm that my changes are working as intended

rktakk committed:
Canada Selected Variables (3) Data only

rktakk commented:
@YMakushenko Hi Yuliia, please see the updated file.

YMakushenko commented:
@rktakk Could you please redo this file? It should contain the following headers:
Year, Population growth (annual %), Fertility rate, total (births per woman), Net migration.    It's the row 3.

You can remove rows 1, 2 and 4.
And you can also delete those rows (years) that don't contain all the data, for example (1960, 2022, 2023 years).

rktakk committed:
Reuploading trimmed dataset

- Removed indicator codes
- Removed years with missing values from the data set (1960, 2022 & 2023)

rktakk committed:
Delete data/processed/Trimmed-dataset-API_8_DS2_en_excel_v2_580868 copy.csv

Removing file as a new one has been uploaded

rktakk commented:
@YMakushenko See new uploaded file.
